docs/1-terms/S/term-service.adoc

```diff
@@ -2,16 +2,28 @@
 
 // tag::EN[]
 ==== Service
-t.b.d.
 
+A service is a technical, autonomous unit that bundles related functionalities, typically on a topic, and makes them available for use by other <term-building-block, building blocks>> via a well-defined interface.
+
+A service optimally abstracts the internal, technical function to such an extent that it is not necessary to know or even understand internal implementation details in order to use the service.
+
+Typical examples of externally accessible services are web services, network services, system services or telecommunications services.
+
+(based on [Wikipedia](https://de.m.wikipedia.org/wiki/Dienst_(Informatik))
```

Comment:
Should be the English URL, right?  🙂 
-> https://en.wikipedia.org/wiki/Service_(systems_architecture)

Reply:
To be honest, no. I actually meant the German Wikipedia page. The English description of "service" (https://en.wikipedia.org/wiki/Service_(systems_architecture)) reflects a different, narrower aspect of service.  I consider the German description to be more appropriate, since it is more abstract.

The English description is spread over several subpages, all describing "services" somehow:
* https://en.wikipedia.org/wiki/Service_(systems_architecture)
* https://en.wikipedia.org/wiki/Network_service
* https://en.wikipedia.org/wiki/Web_service

Reply:
Oh, good to know! Then we should keep the German source. ☺️